www/commands/fetch.md

```diff
@@ -21,6 +21,14 @@ response text.
 
 This command saves the result into the `it` variable.
 
+This command triggers the event `hyperscript:beforeFetch`. This event receives the parameters for the command as its `event.detail`. This is useful to globally change the behaviour of fetch, for example to add a custom header to every request, for example adding an `X-CSRFToken` Header for django apps:
```

Comment:
"for example" is repeated. Also, I think there's no need to name-drop Django here since CSRF tokens are a common technique.

Reply:
You're right, I adjusted that.